latest/apps/buildah/function-buildah.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: core.openfunction.io/v1alpha2
+kind: Function
+metadata:
+  name: function-buildah
+spec:
+  version: "v1.0.0"
+  image: "openfunctiondev/sample-buildah-go:latest"
+  imageCredentials:
+    name: push-secret
+  #port: 8080 # default to 8080
+  build:
+    builder: openfunction/buildah:v1.23.1
+    srcRepo:
+      url: "https://github.com/OpenFunction/samples.git"
+      sourceSubPath: "latest/apps/docker-build"
```

Comment:
Maybe `latest/apps/docker-build` could be changed to `latest/apps/with-dockerfile`

Reply:
Done